cwl/osc2hdf.cwl

```diff
@@ -15,11 +15,10 @@ hints:
 inputs:
   in_file:
     type: File
-    label: osc file
-    format: edam:format_2330
+    label: EDAX binary file (.osc) with EBSD data
     doc: |-
-       vendor:
-       instrument:
+       vendor: EDAX Ametek
+       software: old OIM versions
```

Comment:
Can you specify "old" as in years (2010-2020 ?) or versions ?

Reply:
I wanted to, but I just don't know. I have heard that some new versions do not work with that approach anymore but cannot pinpoint it to a precise version number. 
the "old" is meant as a warning, that new versions might not be working.
similarly: the partial conversion implies, that future versions of the code might be better.